docs/production/deployment.md

```diff
@@ -684,6 +684,28 @@ all at once. This decreases the number of 502's served to clients, at
 the cost of slightly increased memory usage, and the possibility that
 different requests will be served by different versions of the code.
 
+#### `s3_memory_cache_size`
+
+Used only when the [S3 storage backend][s3-backend] is in use.
+Controls the in-memory size of the cache _index_; the default is 1MB,
+which is enough to store about 8 thousand entries.
+
+#### `s3_disk_cache_size`
+
+Used only when the [S3 storage backend][s3-backend] is in use.
+Controls the on-disk size of the cache _contents_; the default is
+100MB.
+
+#### `s3_cache_inactive_time`
+
+Used only when the [S3 storage backend][s3-backend] is in use.
+Controls the longest amount of time an entry will be cached; the
+default is 1d. Since the contents of the cache are immutable, this
+serves only as an additional limit on the size of the contents on
+disk.
```

Comment:
What's the thinking on this value? I would think that for a small/low traffic Zulip server, something like 1 week would be better, specifically for the case that you're self-hosting a relatively low uploads traffic Zulip server in some infrastructure that isn't EC2, even though you're using the S3 storage backend (I can imagine that situation happening in real life). And I don't see much downside to having this value be arbitrarily large.